install.ps1

```diff
@@ -0,0 +1,108 @@
+# Installs KoreBuild into a repo, overwriting all previous build scripts
```

Comment:
Sure, but it also helps for new repos, etc.

I do think we probably only need one of the ps1/sh though...